public/app/plugins/datasource/grafana-testdata-datasource/plugin.json

```diff
@@ -24,7 +25,8 @@
     "logos": {
       "small": "img/testdata.svg",
       "large": "img/testdata.svg"
-    }
+    },
+    "version": "%VERSION%"
```

Comment:
This is necessary to substitute the version but causes the following bug in the UI when running the plugin as core:

![Screenshot from 2023-10-04 11-07-57](https://github.com/grafana/grafana/assets/4025665/96d067bd-df2a-49d7-a00e-5a2037e68953)

My plan is to fix that in a separate PR.

Reply:
Fixed in https://github.com/grafana/grafana/pull/75974

Reply:
Somewhat related to https://github.com/grafana/grafana/issues/74606. 

Remember Ryan mentioning versioning related to the importance of being able to include panel plugin version and datasource plugin version in dashboard queries etc, but that's obviously out of scope for this PR/testdata decoupling but we need to start think about this related to these changes. Think you can create a new issue to track this?

Reply:
I don't think I have enough context for that specific issue. Seems quite unrelated?

 - #74606 is related to the grafana dependency, not the plugin version
 - You are talking about showing plugin versions in dashboards, which seems like a UX issue? Not sure why we would want to do that though.
 
 Maybe I am missing something?